lms/static/js/certificates/views/certificate_whitelist_editor.js

```diff
@@ -103,8 +106,8 @@
                             caller.showMessage(response_data.message);
                         }
                         catch(exception){
-                            caller.showMessage("" +
-                                "Server Error, Please refresh the page and try again."
+                            caller.showMessage(
+                                gettext("Server Error, Please refresh the page and try again.")
```

Comment:
This also needs to be escaped.  In case it is not obvious, the point of escaping is because we can't be sure what characters will be contained in the translated text.